### PR TITLE
Update Homepage Meta Title 

### PR DESCRIPTION
**Pull request by rowa97**
Force merge of:
* "netlify/settings.yaml"

### CODE REVIEW

File: netlify/settings.yaml [hunk 1]
```diff
@@ -187,15 +187,15 @@ footer:
         - fab
         - github
 meta:
-  description: The Complete API Toolkit for Teams and Enterprise | Stoplight
```
Comment (lottamus) [inline netlify/settings.yaml]: Any reason to remove the description?

Comment (rowa97) [reply netlify/settings.yaml]: we don't want it duplicated, but no one has gotten back to me on what the description would be so just pushing title change for now

Comment (lottamus) [reply netlify/settings.yaml]: Ah what do you mean duplicated? like it should only appear on one page?

These meta tags in the settings file are set up to be the fallback for any page that doesnt have them specified

Comment (rowa97) [reply netlify/settings.yaml]: ohhhh, I thought it was only for the homepage, by duplication I mean the meta title and the meta description being the same

Comment (lottamus) [reply netlify/settings.yaml]: Homepage meta is at the bottom: https://develop--stoplightio.netlify.com/admin/#/collections/pages/entries/home

Comment (rowa97) [reply netlify/settings.yaml]: okay so going to submit another pull request, reverting back the site settings changes and updating the homepage settings

**Pull request closed without merging**